software/17.Kubernetes/SETUP.md

```diff
@@ -0,0 +1,31 @@
+# Workshop 17 - Introduction to Kubernetes Setup Guide
+
+This guide contains all the necessary steps to install **Kubernetes** on your machine.
+
+## Installing `Docker`
+
+You can install `Docker` using the [official guide](https://docs.docker.com/get-docker/) or by running the [official convenience installation script](https://get.docker.com/).
+
+If your installation was successful, you should be able to run the following command without any issues.
+
+```
+docker run hello-world
+```
+
```

Comment:
You should propose a [Podman](https://podman.io/) installation if there is a problem with docker.
You should also refactor the step to installing docker according to the [docker workshop](https://github.com/PoCInnovation/Workshops/blob/master/software/04.Docker/SETUP.md)